api/bin/deal-observer-api.js

```diff
@@ -11,15 +9,10 @@ const {
 } = process.env
 
 const pgPool = await createPgPool()
-const logger = {
-  error: console.error,
-  info: console.info,
-  request: ['1', 'true'].includes(REQUEST_LOGGING) ? console.info : () => {}
-}
 
-const handler = createHandler({ pgPool, logger })
-const server = http.createServer(handler)
+const app = createApp({
+  pgPool,
+  logger: ['1', 'true'].includes(REQUEST_LOGGING)
+})
 console.log('Starting the http server on host %j port %s', HOST, PORT)
-server.listen(Number(PORT), HOST)
-await once(server, 'listening')
-console.log(`http://${HOST}:${PORT}`)
+console.log(await app.listen({ host: HOST, port: Number(PORT) }))
```

Comment:
This line does two things - it starts the server and logs some info, presumably about the base URL. I prefer to split it into two lines, if you don't mind.

```suggestion
const serverUrl = await app.listen({ host: HOST, port: Number(PORT) })
console.log(serverUrl)
```